docs/source/markdown/podman-secret-create.1.md

```diff
@@ -78,9 +78,18 @@ delete =
 
 ## EXAMPLES
 
+Create the secret my_secret based on the secret.json file
```

Comment:
"based on" doesn't tell me much. The man page doesn't document any json format. Is json used here in the sense of "the secret will be a json string", or "podman will interpret the json file and extract a part of it according to a magic format"? I'm guessing the former, in which case I'd suggest:
>Create a new secret. Its value will be the full contents of the file secret.json

Reply:
Looks bogus.